### DISCUSSION TITLE
Site "Cursos frontend"

### DISCUSSION THREAD

**Discussion by tarciozemel** [frontendbr/forum#938] [Dicas]
Fala, moçada!

Para quem quiser aproveitar essa semana do djabo para estudar webdev, gostaria de recomendar o site http://www.cursosfrontend.com.br/, lançado recentemente com a ideia de ser uma "lista premium" com os melhores cursos da área em português.

A ideia é incrementar essa lista com o tempo, mas mesmo jovenzinho, já conta com alguns cursos bem interessantes -- inclusive de alguns membros aqui da comunidade.

Att

PS: Ah, e quem quiser fazer sugestões de design/UX, fique à vontade. :grin:

**Comment by felipefialho** [comment]
Não conhecia, maneiro :)

**Comment by lazincastro** [comment]
Boa dica.

**Comment by mahenrique94** [comment]
Boa @tarciozemel \o/

**Comment by MathCunhaDev** [comment]
Boa @tarciozemel Vlw o/

**Comment by marcusvcribeiro** [comment]
Achei até interessante a ideia do site, porem vc tem que pensar no layout ali.
Como que será quando tiver muitos cursos? será um slide infinito?
E já tem cursos de backend la, saindo um pouco do foco do site, entretanto a ideia é legal :+1:

**Comment by tarciozemel** [comment]
@marcusvcribeiro É uma verão inicial somente. No futuro, provavelmente eu vá colocar uma página específica para cada curso para ajudar no conteúdo/SEO.

Questão dos cursos, você deve estar falando do Node, hehe, é controverso mesmo, mas creio que é de grande ajuda um "frontender" saber. Por exemplo, não descarto a possibilidade de colocar um de docker... Enfim, são somente ideias. Qualquer sugestão é bem vinda. 👍
